### DISCUSSION TITLE
How to remove the PlaySessionId field in video stream link?

### DISCUSSION THREAD

**Discussion by niannianyouyu** [jellyfin/jellyfin#9224] [Q&A]
Hi,

I'd like to remove the `PlaySessionId` field in video stream link?
(e.g: http://192.168.x.x:8096/video/..../stream.mp4?PlaySessionId=....&amp;DeviceId=...&amp;MediaSourceId=..., we change it to http://192.168.x.x:8096/video/..../stream.mp4?....&amp;DeviceId=...&amp;MediaSourceId=...)

I have tried to modify the `main.jellyfin.bundle.js` in C:\Program Files\Jellyfin\Server\jellyfin-web, to let the function containing the `PlaySessionId` field return null, but the `PlaySessionId` field still exists in stream link. How should I do?

**Comment by nielsvanvelzen** [comment]
Why do you want to remove this parameter? The server needs it to properly track playback progress.

**Reply by niannianyouyu** [reply]
Because when I remove this field, the link is not related to time, I want my video player (Kmplayer) can remember the stream video, so that I can open it at the last location, or at the location of a customized bookmark in my video player (Kmplayer support bookmark function).

**Reply by niannianyouyu** [reply]
@nielsvanvelzen Can you give me some suggestions, I just want to try it locally😊.
